azerothcore/azerothcore-wotlk
# fix(Scripts/ICC): Don't show Heroic Attempts on Normal difficulty

## CHANGES PROPOSED:
-  Add a IsHeroic check to make sure that you do not get the 50 attempts on Normal.
-  Fix some codestyle


## TESTS PERFORMED:
- Tested in-game




## HOW TO TEST THE CHANGES:

1. .tel icecrowncitadel
2. Change difficulty to Normal/Heroic
3. Inside use
```
.i setbossstate 0 3
.i setbossstate 1 3
.i setbossstate 2 3
.i setbossstate 3 3
.i setbossstate 4 3
```
4. Teleport to deathbringer and run right inside
5. run to council, trigger and kill
6. go to BQ room and see the Heroic Attempts text on HC and not on NM
![image](https://user-images.githubusercontent.com/24550914/91087538-e27f7f00-e650-11ea-8fae-93943cbfff15.png)



## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
